barretenberg/cpp/src/barretenberg/honk/transcript/ultra_transcript.test.cpp

```diff
@@ -85,29 +84,27 @@ class UltraTranscriptTests : public ::testing::Test {
 
         round++;
         // TODO(Mara): Make testing more flavor agnostic so we can test this with all flavors
-        if constexpr (proof_system::IsGrumpkinFlavor<Flavor>) {
-            manifest_expected.add_entry(round, "IPA:poly_degree", size_uint64);
-            manifest_expected.add_challenge(round, "IPA:generator_challenge");
-
-            for (size_t i = 0; i < log_n; i++) {
-                round++;
-                std::string idx = std::to_string(i);
-                manifest_expected.add_entry(round, "IPA:L_" + idx, size_G);
-                manifest_expected.add_entry(round, "IPA:R_" + idx, size_G);
-                std::string label = "IPA:round_challenge_" + idx;
-                manifest_expected.add_challenge(round, label);
-            }
-
-            round++;
-            manifest_expected.add_entry(round, "IPA:a_0", size_FF);
-        } else {
-            manifest_expected.add_entry(round, "ZM:PI", size_G);
-        }
-
+        manifest_expected.add_entry(round, "ZM:PI", size_G);
         manifest_expected.add_challenge(round); // no challenge
 
         return manifest_expected;
     }
+
+    void generate_test_circuit(auto& builder)
+    {
+        FF a = 1;
+        builder.add_variable(a);
+        builder.add_public_variable(a);
+    }
+
+    void generate_random_test_circuit(auto& builder)
```

Comment:
probably should just use this function instead of the other function in all places